app/errors.py

```diff
@@ -0,0 +1,14 @@
+class VaccineError(Exception):
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    pass
```

Comment:
The `OutdatedVaccineError` class currently uses `pass`. Consider adding a docstring or a custom message to provide more context about this error.